helm-charts/seldon-single-model/templates/seldondeployment.json

```diff
@@ -6,15 +6,15 @@
     "namespace": "{{ .Release.Namespace }}",
     "labels": {
       {{- range $name, $val := .Values.labels }}
-        "{{ $name }}": "{{ $val }}"
+        "{{ $name }}": "{{ $val }}",
```

Comment:
Looks good, however with javascript wouldn't you still have an issue if you have a trailing slash? I have seen that with arrays you can use {{if ne $index 0}},{{end}} but in this case you only have name,val so not sure whether it'd be the case. Did it not error out in your case with the extra comma at the end?

Reply:
Helm usually reads and writes YAMLs, and it seems that the YAML parser is happy with trailing commas.